### PR TITLE
DOCSP-34858 Select a cluster option for new sync job

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Add the select a cluster option when creating a sync job

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/docs-relational-migrator/DOCSP-34858/jobs/creating-jobs/#steps

## JIRA
https://jira.mongodb.org/browse/DOCSP-34858?filter=-1

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=662a7c36b67dbf804a344f63

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/jobs/creating-jobs.txt [hunk 1]
```diff
@@ -96,16 +96,43 @@ Steps
 
   .. step:: Enter the source MongoDB connection details 
 
-      On the :guilabel:`Connect Destination DB` form, input your 
-      :guilabel:`MongoDB connection string (URI)`.
+      You can provide the MongoDB connection details by selecting an Atlas
```
Comment (jason-price-mongodb) [inline source/jobs/creating-jobs.txt:99]: Consider rewrite in form "To provide XXX, select ...."

File: source/jobs/creating-jobs.txt [hunk 2]
```diff
@@ -96,16 +96,43 @@ Steps
...
+      cluster or providing a MongoDB connection string (URI). If you are :ref:`logged
```
Comment (jason-price-mongodb) [inline source/jobs/creating-jobs.txt:100]: "If you're ..."

File: source/jobs/creating-jobs.txt [hunk 3]
```diff
@@ -96,16 +96,43 @@ Steps
 
   .. step:: Enter the source MongoDB connection details 
 
-      On the :guilabel:`Connect Destination DB` form, input your 
-      :guilabel:`MongoDB connection string (URI)`.
+      You can provide the MongoDB connection details by selecting an Atlas
+      cluster or providing a MongoDB connection string (URI). If you are :ref:`logged 
+      in with Atlas <rm-atlas-log-in>`, the default option is to :guilabel:`Select 
+      a cluster`. If you are not logged in with Atlas, the :guilabel:`Select a cluster`
```
Comment (jason-price-mongodb) [inline source/jobs/creating-jobs.txt:102]: "If you aren't ..."

**Review by jason-price-mongodb — APPROVED** [review]
@jocelyn-mendez1 lgtm with suggestions. Cheers.

File: source/jobs/creating-jobs.txt (hunk 1 above)
Comment (jocelyn-mendez1) [reply source/jobs/creating-jobs.txt]: Updated.

**Review by alwaysbemark — APPROVED** [review]

**Pull request merged**